app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, favourite_dish: str, nickname: str) -> None:
+        super().__init__(nickname)
+        self.__favourite_dish = favourite_dish
```

Comment:
why do u use private attributes here? Fix everywhere and learn the difference between private and protected ones